src/shadowcasting.h

```diff
@@ -27,6 +27,12 @@ enum class quadrant : int {
     default_ = NE
 };
 
+enum class vertical_direction {
+    UP = 1,
+    DOWN = 2,
+    BOTH = 3
```

Comment:
This is no longer being bitwise OR'd so there's no need to explicitly specify the values